File: modules/animated-counter/ios/AnimatedCounterModule.swift
import ExpoModulesCore

public class AnimatedCounterModule: Module {
  public func definition() -> ModuleDefinition {
    Name("AnimatedCounter")

    View(AnimatedCounterView.self) {
      Prop("count") { (view: AnimatedCounterView, prop: Int) in
        view.count = prop
      }
    }
  } 
}

File: modules/animated-counter/android/src/main/java/expo/modules/animatedcounter/AnimatedCounterModule.kt
package expo.modules.animatedcounter

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class AnimatedCounterModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("AnimatedCounter")

    View(AnimatedCounterView::class) {
      Prop("count") { view: AnimatedCounterView, count: Int ->
        view.updateCount(count)
      }
    }
  }
}
